### PR TITLE
Support services for @typedef

### LINKED ISSUE #15855: Services for @typedef
**TypeScript Version:**  nightly (2.4.0-dev.20170515)

**Code**

```ts
///<reference path="fourslash.ts" />

/////**
//// * @typedef /*def*/Product
//// * @property {string} title
//// */

/////**
//// * @type {/*use*/Product}
//// */
////const product = null

verify.goToDefinition("use", "def");
```

**Expected behavior:**

Get services.

**Actual behavior:**

No services.

### PR DESCRIPTION
**Pull request by andy-ms**
Fixes #15855

### CODE REVIEW

File: src/compiler/utilities.ts [hunk 1]
```diff
@@ -1551,6 +1559,10 @@ namespace ts {
     }
 
    export function getJSDocs(node: Node): (JSDoc | JSDocTag)[] {
+        if (isJSDocTypedefTag(node) && isJSDoc(node.parent)) {
```
Comment (mhegazy) [inline src/compiler/utilities.ts]: Why is this needed? it is not clear what code path would get us here from the change.

Comment (andy-ms) [inline src/compiler/utilities.ts]: quickInfo needs to get the jsdoc to show the doc comment.

Comment (mhegazy) [inline src/compiler/utilities.ts]: i see.. thanks
